go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -25,7 +23,14 @@ const (
 	defaultOpenFlowCookie = "0xdeff105"
 )
 
-func addService(service *kapi.Service, inport, outport, gwBridge string, nodeIP *net.IPNet) {
+type nodePortWatcher struct {
```

Comment:
maybe some doc text around what nodePortWatcher is to make it more clear?

There's a good comment inside the function:
```
// In the shared gateway mode, the NodePort service is handled by the OpenFlow flows configured
	// on the OVS bridge in the host. These flows act only on the packets coming in from outside
	// of the node. If someone on the node is trying to access the NodePort service, those packets
	// will not be processed by the OpenFlow flows, so we need to add iptable rules that DNATs the
	// NodePortIP:NodePort to ClusterServiceIP:Port.
```

Reply:
done